pkg/insights/insightsclient/insightsclient.go

```diff
@@ -187,16 +187,16 @@ func (c *Client) Send(ctx context.Context, endpoint string, source Source) error
 		h.Set("Content-Type", source.Type)
 		fw, err := mw.CreatePart(h)
 		if err != nil {
-			pw.CloseWithError(err)
+			_ = pw.CloseWithError(err)
 			return
 		}
 		r := &LimitedReader{R: source.Contents, N: c.maxBytes}
 		n, err := io.Copy(fw, r)
 		bytesRead = n
 		if err != nil {
-			pw.CloseWithError(err)
+			_ = pw.CloseWithError(err)
 		}
-		pw.CloseWithError(mw.Close())
+		_ = pw.CloseWithError(mw.Close())
```

Comment:
Just a cosmetic change, but shouldn't we handle the error?

Reply:
as far as I know, its ok to not handle errors, but you should explicitly declare that you are not doing it.
in the original the fact that an error can happen is hidden which quite scary, here its clear that it was decided to not handle such an error

Reply:
ok ok. I was looking at https://golang.org/pkg/io/#PipeWriter.CloseWithError and it's probably ok.

Reply:
What @0sewa0 said. When you chose to commit an error handling it must be explicit in the code, otherwise, the linter (errcheck) will consider that you not verified the error/not know about its possibility, and leaving an error without being handled is, in some cases, a potential bug. So, it is not cosmetic, also, I must say that I didn't pay that much attention to the `insightsclient` part of the code, 'cause if I remember we gonna replace it with an external dependency. Isn't that correct?